doc/nrf/app_dev/device_guides/nrf54l/building_nrf54l.rst

```diff
@@ -7,6 +7,10 @@ Building and programming with nRF54L15 DK
    :local:
    :depth: 2
 
+.. note::
+   The FLPR core support in the |NCS| is currently experimental.
+   Enhanced support and an improved developer experience will be provided in future releases.
```

Comment:
I find the second sentence already implied in the first one, so in my opinion it is not really needed.